index.d.ts

```diff
@@ -1,2 +1,2 @@
-declare function loadClass(schema: any, target: any): void;
+declare function loadClass(schema: any, {target: any,options: object}:object): void;
```

Comment:
Will this work when developer uses it without options, like `loadClass(schema, UserModel);`?